contracts/minter/src/contract_tests.rs

```diff
@@ -293,10 +301,22 @@ fn initialization() {
 #[test]
 fn happy_path() {
     let mut router = mock_app();
+    let _timestamp = setup_block_time(&mut router, GENESIS_MINT_START_TIME + 1).unwrap();
```

Comment:
Can just remove `_timestamp` right? No big deal though. Developer decision. It's a test :)

Reply:
yep, cleaned it up!